scripts/build_developer.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e  # Fail on any error.
+set -x  # Display commands being run.
+
+# Change to pytorch directory.
+cd "$(dirname "$(readlink -f "$0")")"
+cd ../../
+
+python3 setup.py bdist_wheel
+python3 setup.py install
```

Comment:
Can you install the package produced by the last line directly? ie `pip install dist/*.whl`

Reply:
I'll keep this the same as the CONTRIBUTING.md for now: https://github.com/pytorch/xla/blob/master/CONTRIBUTING.md, which has worked for people for a while.